src/domain/event/EventUtils.ts

```diff
@@ -281,17 +291,35 @@ const getOfferInfoUrl = (
   return getLocalisedString(offer?.infoUrl, locale);
 };
 
+export const getEventAudience = (
+  event: EventFieldsFragment,
+  locale: Language
+): KeywordOption[] => {
+  return event.audience
+    .map((audience) => ({
+      id: audience.id || '',
+      name: capitalize(audience.name?.[locale] || '').trim(),
+    }))
+    .filter(
+      (audience, index, arr) =>
+        !!audience.id &&
+        !!audience.name &&
+        !EVENT_KEYWORD_BLACK_LIST.includes(audience.id) &&
+        arr.findIndex(
+          (item) => item.name.toLowerCase() === audience.name.toLowerCase()
```

Comment:
no need to make lower case if all item names are capitalized? (line 301)

Reply:
well, true, but do we want to delete them really? I think these should be left here for safety reasons if for some reason capitalization was moved somewhere else or deleted? If capitalization was deleted, you wouldn't notice that toLowerCase needs to be added

Reply:
but of course good tests would catch that error

Reply:
Trying to figure out where this function event came from to this pull request.. I don't remember writing this code at all

Reply:
I think it's better to add unit tests (for upper, lower etc) and then simplify code. Unittest it the best safety guard imho.

Reply:
Well, I agree that it is better to add unit tests anyway. But I wouldn't rely 100% the tests to be correct. .toLowerCase() is not that complicated, just a little verbose

Reply:
refactored + improved tests and decided to get rid of toLowerCase